nautobot/docs/user-guide/administration/tools/nautobot-shell.md

```diff
@@ -213,18 +213,3 @@ To delete multiple objects at once, call `delete()` on a filtered queryset. It's
 !!! warning
     Deletions are immediate and irreversible. Always consider the impact of deleting objects carefully before calling `delete()` on an instance or queryset.
 
-## Change Logging and Webhooks
```

Comment:
I appreciate the additions to the other docs files but I think there's something to be said in favor of keeping this content here as well.